spec/workers/notification_worker_spec.rb

```diff
@@ -102,6 +102,24 @@ def make_it_perform
       end
     end
 
+    context "when the subscriber list has no tags or links" do
+      before do
+        create(:subscriber_list, document_type: 'travel-advice', tags: {}, links: {})
+      end
+
+      it "sends a bulletin when document_type matches" do
```

Comment:
Also the case where it should trigger more than 1 list (eg "all countries" + "travel advice for spain")